gcloud/streaming/transfer.py

```diff
@@ -35,51 +34,99 @@
 RESUMABLE_UPLOAD = 'resumable'
 
 
+_DEFAULT_CHUNKSIZE = 1 << 20
```

Comment:
I recall commenting elsewhere about using

```python
_DEFAULT_CHUNKSIZE = 1048576  # 1 << 20
```

and that bitshifts seem a little too fancy for the "average" maintainer.

WDYT?